jemalloc/jemalloc.go

```diff
@@ -22,6 +23,8 @@ unsigned long long _get_jemalloc_active() {
 */
 import "C"
 
+// So jemalloc is turned on by default? do we need to specify somewhere that jemalloc library is required
+// And could you explain shortly what benefits we found by using jemalloc
```

Comment:
It resulted in much faster CairoVM execution and allow us to monitor Juno's C memory usage.